src/actors/symbolication.rs

```diff
@@ -1463,14 +1468,18 @@ impl SymbolicationActor {
         let mut unwind_statuses = BTreeMap::new();
         let mut object_features = BTreeMap::new();
         let mut frame_info_map = BTreeMap::new();
+        let mut dif_candidates = BTreeMap::new();
 
         // Go through all the modules in the minidump and build a map of the modules with
         // missing or malformed CFI.  ObjectFileStatus::Found is only added when the file is
         // loaded as it could still be corrupted when reading from the cache.  Usable CFI
         // cache files are added to the frame_info_map.
         for (code_module_id, result) in &cfi_results {
             let cache_file = match result {
-                Ok(x) => x,
+                Ok(x) => {
+                    dif_candidates.insert(*code_module_id, x.candidates().clone());
+                    x
```

Comment:
_nit_: Since you're touching that already, can you give this variable a more speaking name?